src/emmetHelper.ts

```diff
@@ -169,6 +171,10 @@ function getCurrentWord(document: TextDocument, position: Position): string {
 	}
 }
 
+function makeCursorsGorgeous(expandedWord: string): string {
+	return expandedWord.replace(/\$\{\d+\}/g, '|').replace(/\$\{\d+:([^\}]+)\}/g, '_$1_');
```

Comment:
Yep, it is markdown syntax, but I was sure this will be added in the future, but now it's better than nothing, and it would be quite cool where you have variables.